web/cypress/integration/plugin/create-delete-in-drawer-plugin.spec.js

```diff
@@ -40,7 +41,7 @@ context('Delete Plugin List with the Drawer', () => {
     cy.login();
   });
 
-  it('should visit plugin market and enable plugin', function () {
+  it('should visit plugin market and enable plugin then show plugin in order by state)', function () {
```

Comment:
```suggestion
  it('should visit plugin market and enable plugin then show plugin in order by state', function () {
```

It looks like there is an extra bracket. Isn't there an error prompt here?